modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,404 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-simple-kmod-using-crd"]
+= Building and running the simple-kmod SpecialResource for a hub and spoke topology
+
+You can use the Special Resource Operator (SRO) on a hub and spoke deployment in which {rh-rhacm-first} connects a hub cluster to one or more managed clusters.
+This example procedure shows how the Special Resource Operator builds driver containers in the hub. SRO watches hub cluster resources to identify OCP versions for the helm charts that it uses to create resources which it delivers to spokes.
```

Comment:
- Is this line break meant to create a new paragraph? If so, needs one more for a blank line.
- SRO is already defined above.
- SRO needs an article.
- Can't use "OCP" in docs.
```suggestion
This example procedure shows how the SRO builds driver containers in the hub. The SRO watches hub cluster resources to identify {product-title} versions for the helm charts that it uses to create resources which it delivers to spokes.
```

Reply:
Done